### ISSUE TITLE
DHCP entries persist after netboot

### ISSUE THREAD

**Issue by stryan** [cobbler/cobbler#1899]
I semi-recently upgraded my cobbler server from 2.6.x to 2.8.2 and it seems DHCP entries are handled differently now. It used to be that dhcp entries were removed after the netboot flag was unset but this no longer appears the case unless I set the system's network interface to be static. Since sometimes I need to provision servers that will use dhcp later (and to our separate DHCP server) this is undesirable.

Is there anyway to remove non-netbooting DHCP entries?

**Comment by jasonmechler** [comment]
There is a /etc/cobbler/settings option that controls this

# always write DHCP entries, regardless if netboot is enabled
always_write_dhcp_entries: 1

**Comment by SchoolGuy** [comment]
@stryan Can you confirm that @jasonmechler's comment helped so that we can close this one?

**Comment by stryan** [comment]
Unfortunately since I originally made this issue our infrastructure has changed and I can no longer recreate the problem, though I believe I was aware of that setting at the time and it didn't seem to be working. It's possible that there was some other setting I misconfigured that was causing it since it appears to no longer be happening. We've also patched our version of Cobbler and the DHCP components to support UEFI so I can't easily double check. I'll close the ticket though since it's probably no longer an issue.
